source/install/self-managed-changelog.md

```diff
@@ -19,6 +19,7 @@ Latest Mattermost Releases:
 ### Important Upgrade Notes
  - Removed the deprecated Insights feature.
  - Mattermost Boards and various other plugins have transitioned to being fully community supported. See this [forum post](https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669) for more details.
+ - The ``channel_viewed`` websocket was changed to ``multiple_channels_viewed``.
```

Comment:
Let me know what other details might be needed here (e.g. who/what does this impact).

Reply:
```suggestion
 - The ``channel_viewed`` websocket event was changed to ``multiple_channels_viewed``, and is now only triggered for channels that actually have unread messages.
```

Reply:
cc @larkox, is this an accurate summary?

Reply:
That seems right, yes.